src/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+  Relay: require('./circuit/relay'),
+  Dialer: require('./dialer'),
+  multicodec: require('./multicodec'),
+  tag: 'Circuit'
+}
```

Comment:
Same reasons - circuit pretends to be a transport to allow plugin in with swarm/libp2p hence the weird similarities.